homeassistant/components/device_tracker/services.yaml

```diff
@@ -0,0 +1,33 @@
+# Describes the format for available device tracker services
+
+see:
+  description: Control tracked device
+
+  fields:
+    mac:
+      description: MAC address of device
+      example: 'FF:FF:FF:FF:FF:FF'
+
+    dev_id:
+      description: Id of device (find id in tracked_devices.yaml)
```

Comment:
Just curious, what is `tracked_devices.yaml`? Did you mean `known_devices.yaml`?